README.md

```diff
@@ -47,28 +46,32 @@ Like a typical  Kubernetes app, Stock Trader use secrets and ConfigMaps to store
 
 
 1. From a terminal window clone the Github repo that has everything needed to deploy the aggregated Stock Trader app.
-```
+
+   ```
    git clone https://github.com/IBMStockTraderLite/stocktrader.git
    cd stocktrader
 
-```
+   ```
 
 2. Retrieve credentials and other details needed to create secrets and/or ConfigMaps
-```bash
+
+   ```bash
    # Note you must be in the scripts sub folder or this command will fail
    cd scripts   
-   # Your instructor will provide the URL
-   ./setupLab.sh [URL of external storage]
-```
+
+   # SETUPPARAMETER environment variable was prepared in the previous steps
+   ./setupLab.sh $SETUPPARAMETER
```

Comment:
change this to read:

`./setupLab.sh $SETUPURL`

to match changes that I made to my fork of your gist